ql/src/java/org/apache/hadoop/hive/ql/exec/repl/AtlasDumpTask.java

```diff
@@ -79,6 +80,7 @@ public AtlasDumpTask() {
   @Override
   public int execute() {
     try {
+      SecurityUtils.reloginExpiringKeytabUser();
```

Comment:
Should it be more closure to the actual usage. Meaning, if we are doing a retry inside the task(Repl/Ranger/Atlas) somewhere, should it be  closer to where we are actually interacting with external system?

Reply:
Yes its there as part of the retry as well. This is to make sure any file system calls happening before that is taken care of.

Reply:
Should we have it for Atlas retries as well?

Reply:
Added